resource-managers/kubernetes/core/src/test/scala/org/apache/spark/deploy/k8s/features/LocalDirsFeatureStepSuite.scala

```diff
@@ -116,4 +115,28 @@ class LocalDirsFeatureStepSuite extends SparkFunSuite {
         .withValue(defaultLocalDir)
         .build())
   }
+
+  test("local dir on mounted volume") {
```

Comment:
This seems to fail on Jenkins. Could you check this once more? Or, do you need a change on test environment?
```
org.apache.spark.deploy.k8s.features.LocalDirsFeatureStepSuite.local dir on mounted volume
```

Reply:
Updated code and unit test, thanks.